### PR TITLE
Issue #1430 - Editing Limpet properties

### PR DESCRIPTION
**Pull request by pecko**

### CODE REVIEW

**Comment by IanMayo** [comment]
Hey, well done. This seems a really tricky one.

**Comment by pecko** [comment]
Thanks. Agree, it was tricky.

File: org.debrief.limpet_integration/src/org/debrief/limpet_integration/data/StoreWrapper.java [hunk 1]
```diff
@@ -384,7 +390,32 @@ public boolean hasEditor()
...
+          public String getDisplayName()
+          {
+            return getDisplayName();
+          }
+
+          @Override
+          public String getName()
+          {
+            return getName();
+          }
```
Comment (IanMayo) [inline org.debrief.limpet_integration/src/org/debrief/limpet_integration/data/StoreWrapper.java]: hmm, I'm surprised that this doesn't cause a recursive stack overflow - where the methods keep calling themselves.

Just to help my Java skills - can you explain why getName() doesn't keep calling itself?

Comment (pecko) [inline org.debrief.limpet_integration/src/org/debrief/limpet_integration/data/StoreWrapper.java]: It calls the methods from parent class, but you are right. It is confused for review (maybe some Java version too). I will try to made it clearer.

Comment (pecko) [inline org.debrief.limpet_integration/src/org/debrief/limpet_integration/data/StoreWrapper.java]: @IanMayo 

Please see https://github.com/debrief/debrief/issues/1453.

Comment (IanMayo) [inline org.debrief.limpet_integration/src/org/debrief/limpet_integration/data/StoreWrapper.java]: Hmm,
super.getPropertyDescriptors() would call the parent class.

I'm sure getPropertyDescriptors() calls this class.

Comment (pecko) [inline org.debrief.limpet_integration/src/org/debrief/limpet_integration/data/StoreWrapper.java]: You are right. I have removed these methods in #1453. We need ordinal methods from EditorType.

**Pull request merged**